.github/workflows/main.yml

```diff
@@ -15,7 +16,17 @@ on:
 # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners
 
 jobs:
+  linux-dump:
```

Comment:
This job is just for debugging, I will remove it before merging!

Reply:
`main.yml` needs a header and description. Can you add those as part of this change?

Reply:
Done, thanks!